packages/react/src/Title/__tests__/Title.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { render, screen } from "@testing-library/react";
+import { describe, it, expect } from "vitest";
+import { axe } from "jest-axe";
+import { Title } from "../Title.agent";
+
+describe("Title", () => {
+  it("should render children", () => {
+    // Act
+    render(<Title>A title</Title>);
+
+    // Assert
+    expect(
+      screen.getByRole("heading", { name: /A title/, level: 2 }),
+    ).toBeInTheDocument();
+  });
+
+  it("should have default class", () => {
+    // Act
+    render(<Title>A title</Title>);
+
+    // Assert
+    expect(
+      screen.getByRole("heading", { name: /A title/, level: 2 }),
+    ).toHaveClass("af-title", {
+      exact: true,
+    });
+  });
+
+  it("should have custom class", () => {
+    // Act
+    render(<Title className="custom-class">A title</Title>);
+
+    // Assert
+    expect(
+      screen.getByRole("heading", { name: /A title/, level: 2 }),
+    ).toHaveClass("custom-class", {
+      exact: true,
+    });
+  });
+
+  it("should have custom class with modifier", () => {
+    // Act
+    render(
+      <Title className="custom-class" classModifier="modifier">
+        A title
+      </Title>,
+    );
+
+    // Assert
+    expect(
+      screen.getByRole("heading", { name: /A title/, level: 2 }),
+    ).toHaveClass("custom-class custom-class--modifier", {
+      exact: true,
+    });
+  });
+
+  it("should not have classModifier attribute", () => {
+    // Act
+    render(
+      <Title className="custom-class" classModifier="modifier">
+        A title
+      </Title>,
+    );
+
+    // Assert
+    expect(
+      screen.getByRole("heading", { name: /A title/, level: 2 }),
+    ).not.toHaveAttribute("classModifier");
+  });
+
+  it("should have correct heading level", () => {
+    // Act
+    render(<Title heading="h3">A title</Title>);
+
+    // Assert
+    expect(
+      screen.getByRole("heading", { name: /A title/, level: 3 }),
+    ).toBeInTheDocument();
+  });
+
+  it("Accessibility violation <Title/>", async () => {
```

Comment:
question, pourquoi n'a tu pas continue `it should ...` dans le nommage des tests ?
a mon sens
```suggestion
  it("should accessibility violation <Title/>", async () => {
```
serait plus parlant non ?